### ISSUE TITLE
Fine-tuning can't save model

### ISSUE THREAD

**Issue by BlueAnthony** [facebookresearch/llama-recipes#166]
### System Info

OS: Ubuntu 22.04
Pytorch: 2.0.1
Cuda: 11.8
GPU: RTX 3060 12G x1

### Information

- [X] The official example scripts
- [ ] My own modified scripts

### 🐛 Describe the bug

As tutorial, I use "python llama_finetuning.py  --use_peft --peft_method lora --quantization --model_name /patht_of_model_folder/7B --output_dir Path/to/save/PEFT/model"

But, there is no model or anything saved in the output_dir "Path/to/save/PEFT/model".

### Error logs

Then, fine-tuning process shows no any error log.

### Expected behavior

Not sure what would be save in output_dir

**Comment by BlueAnthony** [comment]
The loss is nan, so it won't save model.

**Comment by a871545252** [comment]
What means is the "nan"?

**Comment by a871545252** [comment]
I was in the same problem.

**Comment by a871545252** [comment]
Goole means: "If the loss function returns NaN values, it means that the model is unable to learn from the training data. This can be due to several reasons, including incorrect hyperparameters, high learning rates, or insufficient training data."

**Comment by a871545252** [comment]
How can I check the learning loss curve?
